pinot-core/src/main/java/org/apache/pinot/core/common/datatable/DataTableImplV2.java

```diff
@@ -36,6 +35,10 @@
 import org.apache.pinot.spi.utils.ByteArray;
 import org.apache.pinot.spi.utils.BytesUtils;
 
+import static org.apache.pinot.core.common.datatable.DataTableUtils.decodeString;
+import static org.apache.pinot.core.common.datatable.DataTableUtils.deserializeDictionaryMap;
+import static org.apache.pinot.core.common.datatable.DataTableUtils.serializeDictionaryMap;
+
 
 public class DataTableImplV2 implements DataTable {
   private static final int VERSION = 2;
```

Comment:
This now becomes redundant and we can remove this. Use VERSION_2 constant already defined in DataTableBuilder